examples/records/records.bal

```diff
@@ -15,73 +15,37 @@ type Grades record {
 };
 
 public function main() {
-    // This creates an instance of the `Student` record with a non-empty record literal. All the required non-defaultable
-    // fields must be set when initializing a Student record.
-    Student s1 = { name: "John Doe", age: 17,
-                   grades: {maths: 80, physics: 75, chemistry: 65}};
-    io:println(s1);
-
-    // This fetches the value of a specific field in this record. To access fields, use the dot (.) or the index operator.
-    io:println(s1.name);
-    io:println(s1["name"]);
+    // This creates a `Student` record. Since all the fields are required and none of the fields
+    // have explicit default values assigned to them, values must be specified for all the fields
+    // when creating the record.
+    Student john = {name: "John Doe", age: 17,
+                    grades: {maths: 80, physics: 75, chemistry: 65}};
+    io:println(john);
+
+    // This is an example of field-based access of record fields. The return type of this expression is the
+    // type of the field. If it is an open record and the specified key is not present in the record during run time,
+    // it will result in a panic. If it is a closed record, undefined key access will result in a compile error.
```

Comment:
should this be 
```suggestion
    // it will result in a panic. If it is a closed record, undefined key access will result in a compilation error.
```